CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Added Status Logger
```

Comment:
Nit: could you add this in order?

Reply:
Done. Is there a reason this package seems to go in the reverse order of others?

Reply:
No reason.